app/src/pages/explorer/dataset/Dataset.vue

```diff
@@ -14,6 +14,12 @@
               <md-tooltip> {{ shareToolTip }} </md-tooltip>
               <md-icon>share</md-icon>
             </md-button>
+            <div v-if="isAuth && isAdmin">
```

Comment:
Should we grant edit permissions to both the author (i.e., submitter) and administrators? Currently, only admin users have this capability, but what if the author or submitter wishes to make a quick edit, such as fixing a typo?